rhine-terminal/src/FRP/Rhine/Terminal.hs

```diff
@@ -0,0 +1,118 @@
+{- | Wrapper to write @terminal@ applications in Rhine, using concurrency.
+-}
+
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE RecordWildCards #-}
+module FRP.Rhine.Terminal where
```

Comment:
I've goaded you into deleting the export list here. Maybe this was premature. There are things like `terminalT` that we certainly don't want to export. There are two ways to achieve that. One is an export list. The other one is a module `FRP.Rhine.Terminal.Internal` that contains all definitions that should not be exported. I'm fine with both variants, but we should pick one to hide at least `terminalT`.

Reply:
I opted for the explicit export list 👍